packages/core/src/routes/experience/classes/verifications/social-verification.ts

```diff
@@ -480,10 +486,14 @@ export class SocialVerification implements IdentifierVerificationRecord<Verifica
   ) {
     const connector = await this.getConnectorData();
 
+    const isGoogleOneTap = isGoogleOneTapChecker(connectorData);
```

Comment:
Please note this `verifySocialIdentity`  method is still in dev guard.  On production, we still use the `verifySocialIdentity`  method from interaction utils. See line 184.

Reply:
We aligned all logics for both dev feature and prod logic, which should be good.